AK/Concepts.h

```diff
@@ -32,6 +33,9 @@ concept Enum = IsEnum<T>;
 template<typename T, typename U>
 concept SameAs = IsSame<T, U>;
 
+template<typename T>
+concept AnyString = IsSame<String, RemoveCVReference<T>> || IsSame<StringView, RemoveCVReference<T>> || Detail::IsPointerOfType<char, T>;
```

Comment:
How about defining this as "a StringView can be constructed from it"?

Reply:
this would fail on a ByteBuffer-keyed HashTable, but I see your point,
Is excluding ByteBuffers a valid option?
(Also initializer-lists....)

Reply:
Not sure why we need to exclude bytebuffers, but "can be _implicitly_ constructed" would probably fix that.

If that works with initialiser lists (which is something I really doubt), what's the harm? Why say no to `.find({ foo, 123 })` but yes to `.find(foo)`?

Reply:
ByteBuffers don't have a hash function associated with them, but that might never come up